docs/tutorials/13_trotterQRTE.ipynb

```diff
@@ -0,0 +1,983 @@
+    "\n",
+    "$$\n",
+    "\\vert \\psi(t) \\rangle \\approx \\left(\\prod_j e^{-i a_j H_j t/r} \\right)^r \\vert\\psi(0) \\rangle \\text{,}\n",
+    "$$\n",
+    "\n",
+    "where $t/r$ is the time slice (evolution step) that we are choosing. As a result, a gate to be applied $r$ times is created. A smaller timestep leads to a more accurate approximation. However, this also leads to deeper circuits which, in practice, leads to more error accumulation, becoming a critical aspect especially on NISQ devices. For the purposes of this tutorial, all simulations are Statevector simulator, which doesn't include any shot or hardware noise.\n",
+    "\n",
+    "In this tutorial, we will implement real time evolutions using the `TrotterQRTE` class. To illustrate this, we will study the time evolution of the [Ising model](https://en.wikipedia.org/wiki/Ising_model) on linear lattices of $L=2$ and $L=6$ sites. These lattices consist of an array of spins $\\sigma_i$ that interact only with their nearest neighbors. These spins can have two orientations: $\\uparrow$ and $\\downarrow$, which correspond to a magnetization of $+1$ and $-1$ respectively.\n",
+    "\n",
+    "$$\n",
+    "H = - J \\sum_{i=0}^{L-2} \\sigma_i \\sigma_{i+1} - h \\sum_{i=0}^{L-1} \\sigma_i  \\text{,}\n",
+    "$$\n",
+    "\n",
+    "where $J$ describes the interaction energy, and $h$ the magnitude of an external field. Let us write this expression using Pauli matrices, and considering that the external field has an angle $\\alpha$ with respect to the transversal direction,\n",
+    "\n",
+    "$$\n",
+    "H = -J \\sum_{i=0}^{L-2} Z_i Z_{i+1} -h \\sum_{i=0}^{L-1} (\\sin\\alpha Z_i + \\cos\\alpha X_i) \\text{.}\n",
+    "$$\n",
+    "\n",
+    "In the computational basis, the system will be encoded as follows:\n",
+    "\n",
+    "| Quantum state            | Spin representation                        |\n",
+    "|:------------------------:|:------------------------------------------:|\n",
+    "| $\\lvert 0 0 0 0 \\rangle$ | $\\uparrow\\uparrow\\uparrow\\uparrow$         |\n",
+    "| $\\lvert 1 0 0 0 \\rangle$ | $\\downarrow\\uparrow\\uparrow\\uparrow$       |\n",
+    "| $\\ldots$                 | $\\ldots$                                   |\n",
+    "| $\\lvert 1 1 1 1 \\rangle$ | $\\downarrow\\downarrow\\downarrow\\downarrow$ |\n",
+    "\n",
+    "First, we will create a function that takes in the system parameters $L$, $J$, $h$ and $\\alpha$, and returns our Hamiltonian as a `SparsePauliOp`. A [`SparsePauliOp`](https://qiskit.org/documentation/stubs/qiskit.quantum_info.SparsePauliOp.html) is a sparse representation of an operator in terms of weighted [`Pauli`](https://qiskit.org/documentation/stubs/qiskit.quantum_info.Pauli.html) terms."
```

Comment:
```suggestion
    "First, we will create a function that takes in the system parameters $L$, $J$, $h$ and $\\alpha$, and returns our Hamiltonian as a `SparsePauliOp`. A [SparsePauliOp](https://qiskit.org/documentation/stubs/qiskit.quantum_info.SparsePauliOp.html) is a sparse representation of an operator in terms of weighted [Pauli](https://qiskit.org/documentation/stubs/qiskit.quantum_info.Pauli.html) terms."
```
Otherwise it formats badly
![image](https://github.com/qiskit-community/qiskit-algorithms/assets/40241007/034ed41b-030b-4337-89ad-7e0628546e05)